pillar/mine_functions.sls

```diff
@@ -1,7 +1,9 @@
 mine_functions:
-  control_plane_ips:
-    - mine_function: metalk8s.get_control_plane_ips
-  workload_plane_ips:
-    - mine_function: metalk8s.get_workload_plane_ips
+  control_plane_ip:
+    - mine_function: grains.get
+    - metalk8s:control_plane_ip
```

Comment:
Let's try to keep named arguments as much as possible (YAML syntax could make one think there's a `metalk8s` key with a `control_plane_ip` value):

```suggestion
    - key: metalk8s:control_plane_ip
```

FYI, one can also change the delimiter for traversing the grains dict:

```
    - key: metalk8s/control_plane_ip
    - delimiter: "/"
```

Reply:
Done

Reply:
Sooooo.... no. I wanted to do this, since it's a valid remark. However, when I applied the change, lookups from the mine no longer work. No clue why.

Reply:
Sooooooooooooo... yes. Thanks to a hint from @TeddyAndrieux to use the correct YAML encoding, this now works. Thanks! :+1: